cardano-cli/test/cardano-cli-golden/files/golden/help/conway_governance_drep_delegation-certificate.cli

```diff
@@ -33,6 +33,22 @@ Available options:
                            Filepath of the staking verification key.
   --stake-script-file FILE Filepath of the staking script.
   --stake-address ADDRESS  Target stake address (bech32 format).
+  --combined-drep-verification-key STRING
+                           DRep verification key (Bech32 or hex-encoded).
+  --combined-drep-verification-key-file FILE
+                           Filepath of the DRep verification key.
+  --combined-drep-key-hash HASH
+                           DRep verification key hash (either Bech32-encoded or
+                           hex-encoded). Zero or more occurences of this option
+                           is allowed.
+  --combined-stake-pool-verification-key STRING
```

Comment:
what's the difference between combined-stake-pool and stake-pool keys?

Reply:
It looks like it is to make it clear that this is for a combined certificate, isn't it?